master/buildbot/data/forceschedulers.py

```diff
@@ -24,6 +24,7 @@
 def forceScheduler2Data(sched):
     ret = dict(all_fields=[],
                name=unicode(sched.name),
+               button_name=unicode(sched.buttonName),
```

Comment:
buttonName is already documented in forcescheduler section :)
I will fix the unit tests and update the default value of buttonName attribute